dm/docs/RFCS/20220124_enhance_task_manageability.md

```diff
@@ -0,0 +1,131 @@
+
+### dmctl commands for Source
+
+| Command  | Full Syntax Example                                | Flags                   | Arguments                | Description                                                         |
+|----------|----------------------------------------------------|-------------------------|--------------------------|---------------------------------------------------------------------|
+| create   | `dmctl source create source1.yaml`                 |                         | config-file              | create source with config file.                                     |
+| update   | `dmctl source update source1 source1.yaml`         |                         | source-name, config-file | update a source with config file.                                   |
+| delete   | `dmctl source delete --force source1`              | --force(default: false) | source-name              | delete a source with config file.                                   |
+| get      | `dmctl source get --output="source1.yaml" source1` | --output(default: ".")  | source-name              | show the source config in yaml format, also support output to file. |
+| list     | `dmctl source list`                                |                         |                          | list all source in current cluster.                                 |
+| status   | `dmctl source status source1`                      |                         | source-name              | show source detail status.                                          |
+| enable   | `dmctl source enable source1`                      |                         | source-name              | enable a disabled source.                                           |
+| disable  | `dmctl source disable source1`                     |                         | source-name              | disable a source and also stop the running subtasks of this source. |
+| transfer | `dmctl source transfer source1 worker1`            |                         | source-name, worker-name | transfers a source to a free worker.                                |
+
+### dmctl commands for Relay
+
+| Command | Full Syntax Example                                                                                 | Flags                      | Arguments              | Description                                                                  |
+|---------|-----------------------------------------------------------------------------------------------------|----------------------------|------------------------|------------------------------------------------------------------------------|
+| start   | `dmctl relay start --worker-name="worker1" source1`                                                 | --worker-name(default: "") | source-name            | start relay for a source on a worker.                                        |
```

Comment:
we need to state that if worker-name is empty, we start relay at bound worker of this source

Reply:
also will be add in doc